src/main/webapp/availabilityTable.jsp

```diff
@@ -21,14 +21,42 @@
   <tbody>
     <!-- TODO: Allow clicking and scrolling over multiple slots to select them.-->
     <!-- TODO: Change page format so that it is vertically condensed.-->
-    <c:forEach var = "i" begin = "0" end = "${pageScope.list.get(0).size() - 1}">
+    <tr>
+      <td id="first-slot" onclick="toggleTile(this)" 
+        data-utc="${list.get(0).get(0).utcEncoding()}" 
+        class="${list.get(0).get(0).selected() ? 'table-success' : ''}">
+        ${list.get(0).get(0).time()}
+      </td>
+      <c:forEach var = "i" begin = "1" end = "6">
+        <td onclick="toggleTile(this)" data-utc="${list.get(i).get(0).utcEncoding()}" 
+          class="${list.get(i).get(0).selected() ? 'table-success' : ''}">
```

Comment:
I'm going to break my "dont complain about something without a suggestion rule" this time.

This, and especially the calls below, are a mess. Please refactor this.

Reply:
Fixed.